### PR TITLE
replaced assert false by llvm-unreachable, squash no return warning

### PR DESCRIPTION
**Pull request by AlexandreEichenberger**
Signed-off-by: Alexandre Eichenberger <alexe@us.ibm.com>

minor change, remove one warning by the compiler

### CODE REVIEW

File: utils/OpBuildTable.inc [hunk 1]
```diff
@@ -0,0 +1,354 @@
+//********************************************************
```
Comment (chentong319) [inline utils/OpBuildTable.inc:1]: This file should be move to src/Build

**Review by chentong319 — COMMENTED** [review]
Do not add util/OpBuildTable.inc. It should be in src/Build. In your case, this file is not changed.

**Review by chentong319 — APPROVED** [review]
LGTM!

**Comment by AlexandreEichenberger** [comment]
> Do not add util/OpBuildTable.inc. It should be in src/Build. In your case, this file is not changed.

@chentong319 Thanks, removed the file in util. The one in OpBuildTable.inc is unchanged from master.

**Pull request merged**